src/pages/prescreen-1c.tsx

```diff
@@ -46,103 +45,53 @@ width: 70%;
-      coveredEmployment: null,
-      pensionOrRetirementAccount: null,
-      pensionType: null,
-      pensionAmount: null
-    }
-  }
-
-  componentDidMount() {
-    if (SessionStore.get("dateAwarded") && (this.state.dateAwarded === null)){
-      var dateAwarded = new Date(JSON.parse(SessionStore.get("dateAwarded")))
-      this.setState({
-        dateAwarded: dateAwarded
-      })
-    }
-
-    if (!this.state.isLoaded) {
-      this.setState({
-        isLoaded: true,
-        coveredEmployment:
-          booleanFromString(SessionStore.get("coveredEmployment")),
-        pensionOrRetirementAccount:
-          SessionStore.get("pensionOrRetirementAccount"),
-        pensionType: SessionStore.get("pensionType")
-          ? SessionStore.get("pensionType")
-          : null,
-        pensionAmount: SessionStore.get("pensionAmount")
-          ? SessionStore.get("pensionAmount")
-          : undefined
-      });
-    }
-  }
-  
-  handleDateChange(name, value){
+class Prescreen1c extends React.Component<Prescreen1cProps> {
+  handleDateAwardedChange = (value: Date) => {
+    const {userStateActions: {setPensionDateAwarded}} = this.props
     if (name === "dateAwardedPicked") {
-      SessionStore.push("dateAwarded", JSON.stringify(value))
-      var state = {dateAwarded: value}
-      this.setState(state)
+      setPensionDateAwarded(value)
     }
   }
-  
-  handleSelection(e) {
-    let selectValueString = e.target.value;
-    let selectValue;
 
+  handleSelection = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const {
+      userStateActions: {
+        setIsEmploymentCovered,
+        setPensionOrRetirementAccount,
+        setPensionAmount,
+      }
+    } = this.props
+    const selectValueString = e.target.value;
     switch (e.target.name) {
       case "coveredEmployment":
-        selectValue = booleanFromString(selectValueString);
-        SessionStore.push("coveredEmployment", selectValue);
-        this.setState({
-          coveredEmployment: selectValue
-        });
+        const isCovered = selectValueString === 'true'
+        setIsEmploymentCovered(isCovered)
         break;
       case "pensionOrRetirementAccount":
-        SessionStore.push("pensionOrRetirementAccount", selectValueString);
-        this.setState({
-          pensionOrRetirementAccount: selectValueString
-        });
-        break;
-      case "monthlyPension":
-      case "lumpSum":
-        SessionStore.push("pensionType", e.target.name);
-        this.setState({
-          pensionType: e.target.name
-        });
+        setPensionOrRetirementAccount(selectValueString as PensionEnum)
         break;
       case "pensionAmount":
-        SessionStore.push("pensionAmount", e.target.value);
-        this.setState({
-          pensionAmount: e.target.value
-        });
+        const pensionAmount = parseFloat(e.target.value)
+        if (!isNaN(pensionAmount) && pensionAmount > 0) setPensionAmount(pensionAmount)
+        else setPensionAmount(0)
```

Comment:
This changed from empty with a red placeholder:
<img width="532" alt="image" src="https://user-images.githubusercontent.com/283343/73420065-5e9a2a00-42ef-11ea-8669-2a07400243c1.png">

to a zero by default:
<img width="554" alt="image" src="https://user-images.githubusercontent.com/283343/73420015-33173f80-42ef-11ea-9d50-e35f3620223b.png">

Should we handle this as a special case on the front end or do we want to keep it null/undefined until they enter something. 

We want to call out it as strange if they do not put a value in when they said they have some sort of government pension/account.